utils/data.py

```
@@ -0,0 +1,50 @@
+
+
+def read_json(path: Path) -> Any:
+    try:
+        if not path.exists():
+            return []
+
+        with path.open("r", encoding="utf-8") as f:
+            return json.load(f)
+
+    except json.JSONDecodeError as e:
+        logger.error("Corrupted JSON file: %s", path, exc_info=e)
```

Ruff (0.14.11)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)